time stamp_obs_update


# +++++++ observations +++++++++
bool[] real_obs_received
uint32 mapped_observation_pomdp
uint32 mapped_observation_raw 
float32 immediate_reward_IE
float32 discounted_reward_IE 


# ++++++ task and subtask status  +++++  
string who_succeeded


